src/material/core/theming/tests/postcss-compare.ts

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
Could this file (and its corresponding `ts_library`) live somewhere under `tools/` rather than `src/material`? (the test itself is fine here, though)